indexing-service/src/main/java/io/druid/indexing/overlord/RemoteTaskRunner.java

```diff
@@ -128,6 +134,13 @@
 
   private volatile boolean started = false;
 
+  private static final ScheduledExecutorService exec = ScheduledExecutors.fixed(
```

Comment:
Can we rename this cleanerExec or something a little more descriptive (in the thread name template as well)